packages/manager/modules/pci/src/projects/project/quota/quota.constants.js

```diff
@@ -6,9 +6,12 @@ export const RESTRICTED_INSTANCES = 1;
 
 export const VALID_PAYMENTMEAN = 'VALID';
 
+export const RX_PLAN_CODE_PATTERN = /quota-([0-9]+)vms/;
```

Comment:
As mentioned by Sonarcloud, the regex could be simplify as following:
```suggestion
export const RX_PLAN_CODE_PATTERN = /quota-(\d+)vms/;
```